app/helpers/admin/products_helper.rb

```diff
@@ -11,19 +11,20 @@ def product_image_form_path(product)
     end
 
     def prepare_new_variant(product)
-      product.variants.build do |variant|
-        variant.unit_value = 1.0 * (product.variant_unit_scale || 1)
```

Comment:
Mimicking old UI behavior, when creating a new variant, then both of these are empty